Jbarget/oxymore
# So/conscious shopping image carousel

### What changes have you made?
- added forward and back functionality to the conscious shopping carousel 
- the user can navigate forward and backwards using the keyboard also 
- improved responsiveness by creating an alternative layout for desktop and tablet

### Which issue(s) does this PR fix?

Fixes #309 
Fixes #353 

### Screenshots (if there are design changes)
<img width="231" alt="Screenshot 2021-02-18 at 15 35 38" src="https://user-images.githubusercontent.com/53219789/108380880-31246500-71ff-11eb-8860-ce60eced71e5.png">



### How to test
- Set the timer back to a date in the past and the content should appear 
- Test the carousel by navigating forwards and backwards 
- Test the same using keyboard
- Check for a responsive layout on mobile 

### Known bugs 
- There's a page jump when clicking on either of the arrows on small desktop or mobile screens
- It's a bug that's been reproduced on Chrome and Safari 

Update. 

- Unable to find a fully workable solution to the page jump so I've repositioned the arrows on mobile and smaller desktop screens
- This solves the issue on Chrome but not on Safari..
